apps/src/applab/designMode.js

```diff
@@ -197,7 +212,10 @@ designMode.onDonePropertiesButton = function() {
 
 designMode.onDeletePropertiesButton = function(element, event) {
   element.parentNode.removeChild(element);
-  Applab.levelHtml = designMode.serializeToLevelHtml();
+  if ($(element).hasClass('screen')) {
```

Comment:
I think I ideally we'd actually make the delete button disabled in this case (rather than just ignoring clicks to it). That will be done in a future commit.